src/components/SurveyAnswer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { colors, texts } from '../config';
+import { useSurveyLanguages } from '../hooks';
+import { ResponseOption } from '../types';
+import { Radiobutton } from './Radiobutton';
+import { BoldText, RegularText } from './Text';
+import { Touchable } from './Touchable';
+import { Wrapper, WrapperRow } from './Wrapper';
+
+type Props = {
+  faded: boolean;
+  index: number;
+  responseOption: ResponseOption;
+  selected: boolean;
+  setSelection: (id: string) => void;
+};
+
+const getAnswerLabel = (lang: 'de' | 'pl', index: number) => {
+  return `${texts.survey.answerLabelPrefix[lang]} ${String.fromCharCode(65 + index)}:`;
+};
+
+export const SurveyAnswer = ({ faded, index, responseOption, selected, setSelection }: Props) => {
+  const languages = useSurveyLanguages();
+
+  const { id } = responseOption;
+  const onPress = useCallback(() => setSelection(id), [id, setSelection]);
```

Comment:
should it be a part already to not make selections in archived surveys?

Reply:
i dont see that being part of any ticket

Reply:
cool, will you address that in the daily?

Reply:
addressed.

not part of this ticket, but should be done.

will do another PR that adds this logic in the end